Source/SaveExtension/Private/Serialization/SlotDataTask_Saver.cpp

```diff
@@ -117,7 +117,19 @@ void USlotDataTask_Saver::Tick(float DeltaTime)
 	if (SaveInfoTask && SaveDataTask && 
 		SaveInfoTask->IsDone() && SaveDataTask->IsDone())
 	{
-		Finish(true);
+		if (bSaveThumbnail)
+		{
+			USaveManager* Manager = GetManager();
+			USlotInfo* CurrentInfo = Manager->GetCurrentInfo();
+			if (CurrentInfo->GetThumbnail())
```

Comment:
This is interesting. So you wait for the screenshot to be made, good idea.

Doesn't the SlotDataTask already have a pointer to SlotInfo?
You may not need to find it using the Manager.

Reply:
done

Reply:
done